directory/src/App.js

```diff
@@ -16,7 +16,7 @@ class Icon extends PureComponent {
   render() {
     return (
       <span style={{ fontFamily: this.props.family }} {...this.props}>
-        {String.fromCharCode(IconFamilies[this.props.family][this.props.name])}
+        {String.fromCodePoint(IconFamilies[this.props.family][this.props.name])}
```

Comment:
while you are handling this PR, I would suggest refactoring code a little bit.
Let's use js destructuring here (for props)

Reply:
done

Reply:
maybe for this part: `<span style={{ fontFamily: this.props.family }} {...this.props}>` 
better update it to `<span style={{ fontFamily: family }} {...this.props}>` since its already destructured at the top